### PR TITLE
Rollbar integration

### PR DESCRIPTION
**Pull request by anvmn**
#842

### CODE REVIEW

File: client/src/js/app.js [hunk 1]
```diff
@@ -1331,6 +1334,101 @@ function reportSignaturePadResult(url) {
...
+
+elmApp.ports.initRollbar.subscribe(function(data) {
+  // Generate rollbar config.
+  var _rollbarConfig = {
+      accessToken: data.token,
+      captureUncaught: true,
+      captureUnhandledRejections: true,
+      payload: {
+          environment: 'all',
+          // context: 'rollbar/test'
```
Comment (AronNovak) [inline client/src/js/app.js]:
```suggestion
```

File: client/src/js/app.js [hunk 2]
```diff
@@ -1331,6 +1334,101 @@ function reportSignaturePadResult(url) {
...
+          client: {
+            javascript: {
+              code_version: '1.0',
+              // source_map_enabled: true,
+              // guess_uncaught_frames: true
```
Comment (AronNovak) [inline client/src/js/app.js]:
```suggestion
```

File: client/src/js/nodes.js [hunk 3]
```diff
@@ -276,7 +276,29 @@
...
+                                                        status: 400,
+                                                        statusText: 'Failure: PATCH at changes table'
+                                                    });
+
+                                                    return Promise.resolve(reject);
+                                                }).then(function (localId) {
+                                                    return Promise.resolve(response);
+                                                });
+                                            }).then(function (localId) {
+                                                // @todo: remove this
```
Comment (AronNovak) [inline client/src/js/nodes.js]: When will we resolve this todo? Is there a blocker to do it within the PR?

Comment (anvmn) [reply client/src/js/nodes.js]: Yeah, it was added to simulate errors, but we don't need to keep it. Removed.

**Comment by anvmn** [comment]
@AronNovak 

Dropped commented out stuff

**Review by AronNovak — APPROVED** [review]
:+1:

**Pull request merged**